config/environments/development.rb

```diff
@@ -71,8 +71,9 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.hosts << `hostname`.chomp
```

Comment:
please add a comment why this is done here

Reply:
Was for local env, should not have committed this.